plugins/omkafka/omkafka.c

```diff
@@ -1054,6 +1055,9 @@ loadFailedMsgs(instanceData *const __restrict__ pData)
 	uchar *puStr;
 	char *pStrTabPos;
 
+	/* Avoid false positives in CLANG */
```

Comment:
I think this is NOT a false positive. Without the change in line 1476 it could be NULL. So I would remove the comment. The assert itself probably makes sense, albeit I think it really wants to check against ```!= NULL```.